src/ldw-barsPlus.js

```diff
@@ -281,7 +283,8 @@ export default {
             qNum: num,
             qText: txt,
             qElemNumber: elm,
-            offset: t
+            offset: t,
+            mNum: mNum
```

Comment:
when propertyName and value are the same you can use shorthand:
`mNum: mNum` -> `mNum`